features/filesystem/fat/FATFileSystem.cpp

```diff
@@ -142,11 +139,11 @@ DWORD get_fattime(void)
     time(&rawtime);
     struct tm *ptm = localtime(&rawtime);
     return (DWORD)(ptm->tm_year - 80) << 25
-           | (DWORD)(ptm->tm_mon + 1  ) << 21
-           | (DWORD)(ptm->tm_mday     ) << 16
-           | (DWORD)(ptm->tm_hour     ) << 11
-           | (DWORD)(ptm->tm_min      ) << 5
-           | (DWORD)(ptm->tm_sec/2    );
+           | (DWORD)(ptm->tm_mon + 1) << 21
+           | (DWORD)(ptm->tm_mday) << 16
+           | (DWORD)(ptm->tm_hour) << 11
+           | (DWORD)(ptm->tm_min) << 5
+           | (DWORD)(ptm->tm_sec / 2);
```

Comment:
Hm, this just got harder to read. Is it possible to make astyle more lax in this situation?